src/Microsoft.Azure.WebJobs.Host/Executors/ExceptionExtensions.cs

```diff
@@ -1,70 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
Consolidated this into the other `ExceptionExtensions` in the Extensions folder.